HomeTask2/HomeTask2/Program.cs

```diff
@@ -0,0 +1,20 @@
+﻿using System;
+
+namespace HomeTask2
+{
+    class Program
+    {
+        static void Main()
+        {
+            double volume, square;
+            Console.WriteLine("Введите значение длины стороны основания правильной треугольной пирамиды");
+            var side = double.Parse(Console.ReadLine());
+            Console.WriteLine("Введите значение высоты правильной треугольной пирамиды");
+            var height = double.Parse(Console.ReadLine());
+            volume = (height * Math.Sqrt(3) * side * side * 0.25) / 3;
+            Console.WriteLine($"Объем правильной треугольной пирамиды равен {volume}");
+            square = (0.5 * (side + side + side) * Math.Sqrt(height * height + ((side * side * 3) / 36))) + side * side * Math.Sqrt(3) * 0.25;
```

Comment:
Формула верная, но можно было упростить
```suggestion
            square = 1.5 * side * Math.Sqrt(height * height + (side * side / 12)) + side * side * Math.Sqrt(3) * 0.25;
```